main.go

```
@@ -0,0 +1,28 @@
+
+	// Gin 라우터 생성
+	r := gin.Default()
+
+	// 라우트 정의
+	r.POST("/create-name", nameHandler.CreateName)
+	r.GET("/get-names", nameHandler.GetNames)
+	r.DELETE("/delete-index", nameHandler.DeleteByIndex) // 인덱스로 삭제 (기존 /delete-name 변경)
+	r.DELETE("/delete-name", nameHandler.DeleteByName)   // 이름으로 삭제 (신규)
+
+	// 서버 시작
+	r.Run(":8080")
```

golangci-lint (1.64.8)

Error return value of `r.Run` is not checked

(errcheck)